pkg/version/release.go

```diff
@@ -3,7 +3,7 @@ package version
 // This file was generated by release_generate.go; DO NOT EDIT.
 
 // Version is the version number in semver format X.Y.Z
-var Version = "0.6.0"
```

Comment:
don't we have something that bumps this automatically?

Reply:
no. I did look into it as part of https://github.com/weaveworks/pctl/pull/267, but to automate this it would just open a new PR (since we can't  push to main). I honestly felt like it wasn't worth the cost. The current release process is to open a PR that:
- includes the release notes
- bump the version value in `release.go`

and run the release github action.